src/Symfony/Bridge/Doctrine/DataCollector/DoctrineDataCollector.php

```diff
@@ -56,17 +56,14 @@ public function __construct(
      *
      * @deprecated since Symfony 6.4, use a DebugDataHolder instead.
      */
-    public function addLogger(string $name, DebugStack $logger)
+    public function addLogger(string $name, DebugStack $logger): void
```

Comment:
adding a return type on a non-final and non-private method is a BC break. This cannot be done in 6.4